build-tools/mono-runtimes/mono-runtimes.targets

```diff
@@ -8,7 +8,8 @@
   </PropertyGroup>
   <ItemGroup>
     <MonoDocCopyItem Include="monodoc.dll" />
-    <MonoDocCopyItem Include="monodoc.dll.mdb" />
+    <MonoDocCopyItem Include="monodoc.pdb"  Condition="Exists ('monodoc.pdb')" />
```

Comment:
@xmcclure sorry, I completely forgot that we needed the path in there. 

Best thing to do is to write a target which will collect the files that exist. You can use the ItemGroup code that @jonpryor mentioned and pop that in the new Target. That will add the items to the ItemGroup at run time not load time. 

```
<Target Name="_CollectDebuggingFiles">
<ItemGroup>
  <_MonoDocCopyItems
      Condition=" Exists ('$(_MonoOutputDir)\%(Identity)') "
      Include="@(MonoDocCopyItem->'$(_MonoOutputDir)\%(Identity)')"
  />
  <_MonoDocInstalledItems
      Condition=" Exists ('$(_MonoOutputDir)\%(Identity)') "
      Include="@(MonoDocCopyItem->'$(_MandroidDir)\%(Identity)')"
  />
</ItemGroup>
</Target>
```

Then just add the _CollectDebuggingFiles target to the DependsOnTargets list of the target you need it to run before.